WordPress/Sniff.php

```diff
@@ -265,6 +265,8 @@
 		'sanitize_title'             => true,
 		'sanitize_user_field'        => true,
 		'sanitize_user'              => true,
+		'sanitize_hex_color'	     => true,
+		'sanitize_hex_color_no_hash' => true,
```

Comment:
The function names are sorted alphabetically so these functions should be added to line 254.